src/main/java/edu/uc/inventorymanager/dto/Item.java

```diff
@@ -4,7 +4,7 @@
 
 public @Data class Item {
     private int id;
-    private String name;
+    private String itemName;
```

Comment:
According to this article https://www.baeldung.com/java-clean-code ,it’s important for the variable name to be distinctive. The three dto classes had the same variable name for “name”. Even though they were in different classes, I think it’s still important for them to be different for future understanding.